app/components/Users/SignupForm.jsx

```diff
@@ -77,6 +78,11 @@ class RealSignupForm extends React.PureComponent {
         </strong>
         <hr />
         {renderAllUserFields(valid, t)}
+        <P mb="15px" color="#666" fontSize="12px">
```

Comment:
I would recommend using css ans className.

Reply:
At the moment this approach (`styled-components`) is considered valid and is encouraged. We may actually want to deprecate `sass` in the future and pick this one as the default.